mast/processor.py

```diff
@@ -0,0 +1,78 @@
+from math import radians, cos, sin, asin, sqrt
+import xml.etree.ElementTree as ET
+from pathlib import Path
+import os
+from re import findall
+import logging
+
+
+class GPXProcessor(object):
+    EARTH_RADIUS = 6378
+    LANDING_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'landing')
+    XML_SUFFIX = '.xml'
```

Comment:
If it's not a large problem, we want to allow also .gpx suffix

Reply:
Fixed.